notebooks/ch-applications/qaoa.ipynb

```diff
@@ -53,12 +58,13 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## QAOA\n",
     "\n",
-    "QAOA (Quantum Approximate Optimization Algorithm) introduced by Farhi et al.[1] is a quantum algorithm that attempts to \n",
+    "QAOA (Quantum Approximate Optimization Algorithm) was introduced by Farhi et al [1]. is a quantum algorithm that attempts to \n",
```

Comment:
```suggestion
    "QAOA (Quantum Approximate Optimization Algorithm), introduced by Farhi et al. [1], is a quantum algorithm that attempts to \n",
```

Couple of things here:
 - "Et al." needs the period as it's an abbreviation
 - The "introduced by..." is an interjection, so I've removed the "was" and surrounded with commas.

Reply:
My mistake! I was trying to it up into two sentences; I should have reviewed this. My apologies, having the "...introduced by" as an interjection reads better.

Reply:
No worries! Thanks